test/goldens/build_file_web_application

```diff
@@ -28,3 +31,22 @@ dart_web_application(
         ":silly_monkey",
     ],
 )
+dev_server(
+    name = "main_web_dartium_serve",
+    deps = _PUB_DEPS + [
+        ":silly_monkey",
+    ],
+    data = glob(["web/**"]),
+)
+dart_ddc_bundle(
+    name = "main_web_ddc_bundle",
+    entry_library = "web/main.dart",
```

Comment:
Ugh. OK, I can hack it for now by emitting:

```build
dart_library(
  name = ...,
  srcs = glob(["lib/**"]) + glob(["web/**"]),
)
```

Long-term we should just have `dart_web_application` handle this.